### ISSUE TITLE
MacVim sha256 sum for snapshot-77 is incorrect

### ISSUE THREAD

**Issue by rdustinb** [mistydemeo/tigerbrew#1189]
brew update was run prior to the attempt to install MacVim.

brew doctor doesn't report any issues.

Attempting brew install macvim results in the following error:

```bash
Error: SHA256 mismatch
Expected: 6b7f4b48ecef4a00dca17efef551884fcea1aa9897005497d31f52da7304bc5f
Actual: 4696b2cd6f69093c8a619daee9a0db441ce1a124edfddd3b0b3fed3ac96f8e8d
```

Downloading the snapshot file manually and calculating the sha256 hash reveals the correct checksum matches the actual reported by Tigerbrew.

```bash
$ sha256sum snapshot-77.tar.gz 
4696b2cd6f69093c8a619daee9a0db441ce1a124edfddd3b0b3fed3ac96f8e8d  snapshot-77.tar.gz
```

Looking at the Homebrew formula for this release indicates this same issue is present in the Homebrew formula as well. Reference:

https://github.com/Homebrew/homebrew-core/blob/98efc62bcfb7313b5f750a6023df1879dbd2e654/Formula/macvim.rb

Is this an upstream issue in Homebrew or should it be fixed in Tigerbrew?

**Comment by mistydemeo** [comment]
Hmm, interesting. I'm seeing the issue. I don't see a sign that this was retagged, so it's possible that the tarball really has just changed checksum in the interim.

**Comment by mistydemeo** [comment]
If you find that newer versions still build then yes, absolutely - it would be great to get it updated! Are you interested in trying that out?

**Comment by rdustinb** [comment]
I am!

Let me see how far I can update with the current formula and dependency structure before install breaks. I would have to learn a lot more to go further than that.

For now, would you like me to update the current formula with the correct checksum?

**Comment by mistydemeo** [comment]
That would be great, thank you!

**Comment by mistydemeo** [comment]
Resolved by #1190.
